set.mm

```diff
@@ -645962,6 +645970,53 @@ D Fn ( I ... ( M - 1 ) ) /\
     VLVQVLULJVPVLIVRVLUMUNVLVQJZVLNJZKVLORZVLAORZVLVRPABLMQSUOWBAACDZBORWCGBOAU
     PALQUQURUSUTABALMLSQVAVCKNJANJVSVTWAWBVDVEVFALPKAVLVGTVHVIVLVJTVK $.
 
+  ${
+    $d .0. a b $.
+    $( The right conjunct in the right hand side of the equivalence of ~ isdomn
+       is logically equivalent to a less symmetric version where one of the
+       variables is restricted to be nonzero.  (Contributed by SN,
+       16-Sep-2024.) $)
+    isdomn5 $p |- (
+      A. a e. B A. b e. B ( ( a .x. b ) = .0. -> ( a = .0. \/ b = .0. ) ) <->
+      A. a e. ( B \ { .0. } ) A. b e. B ( ( a .x. b ) = .0. -> b = .0. ) ) $=
+      ( cv co wceq wo wi wral wne csn cdif wn bi2.04 df-ne imbi1i df-or imbi2i
+      3bitr4ri 2ralbii r19.21v ralbii raldifsnb 3bitri ) DFZEFZBGCHZUGCHZUHCHZI
+      ZJZEAKDAKUGCLZUIUKJZJZEAKZDAKUNUOEAKZJZDAKURDACMNKUMUPDEAAUJOZUOJUIUTUKJZ
+      JUPUMUTUIUKPUNUTUOUGCQRULVAUIUJUKSTUAUBUQUSDAUNUOEAUCUDURDACUEUF $.
+  $}
```

Comment:
Should this rather be named something like `isdomn4lem` ?
For a theorem `isdomn5`, I would expect it to be of the form ` R e. Domn <-> ... `

Reply:
The comment would still be that this is an equivalence for df-domn, which I think indicates this is not a lemma. It's actually even more useful than the usual form.

Reply:
Good enough for me then!